media/css/pebbles/base/elements/_lists.scss

```diff
@@ -59,3 +63,13 @@ ol.prose {
         }
     }
 }
+
+dl.prose {
+    dt {
+        font-weight: bold;
+    }
+    dd {
+        margin-bottom: .25em;
+        padding-left: 60px;
```

Comment:
Visually, is this maybe a little much?

<img width="866" alt="screen shot 2018-03-15 at 14 37 48" src="https://user-images.githubusercontent.com/400117/37470009-79cdec6a-285e-11e8-9b1d-1348deae5962.png">

Reply:
Decreased to `$gutter-width`